metrics/density/footprint_data.md

```diff
@@ -0,0 +1,108 @@
+# Footprint data script details
+
+The `footprint_data.sh` script sequentially runs a number of identical containers
+via the docker engine and takes a number of memory related measurements after each
+launch. The script is generally not used in a CI type environment, but is intended
+more for hand run/analysis.
+
+The script can be configured via a number of environment variables.
+
+Details of the configurable variables are listed below, along with a small example
+invocation script.
+
+## Variables
+The script action can be modified by setting a number of environment variables.
```

Comment:
Replace line 14 and the beginning of line 16 with: "Environment variables can take effect in two ways." Add a blank line after that then resume with "Some variables ..."